specification/compute/resource-manager/Microsoft.Compute/stable/2021-07-01/gallery.json

```diff
@@ -2120,6 +2120,18 @@
         "purchasePlan": {
           "$ref": "#/definitions/ImagePurchasePlan"
         },
+        "architecture": {
```

Comment:
please make this the last subproperty of its parent property

Reply:
Done